### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by katznd** [community/community#17911] [Other Feature Feedback, Questions, & Ideas] [answered]
hy , my question is how can i delete my profile delete account is this possible?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - this space is for Product Feedback on GitHub. To delete your account, I'd recommend checking out our Docs like this one [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/deleting-your-personal-account).
